frontend/src/index.pug

```diff
@@ -34,6 +34,14 @@ html
     script(src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.14/build/languages/yaml.min.js")
   body
     #app
+      loading-overlay.overlay-loader(
+        :active.sync="isLoadingOverlayEnabled",
```

Comment:
should we stick to the same `v-bind` format as the rest instead of the shorthand for consistencies' sake?

Reply:
Yep, better to be consistent.